File: Java/Ejercicio1.java
import java.util.ArrayList;

public class Ejercicio1 {
    public static void main(String[] args) {
        // Crear un ArrayList de String con 5 nombres de estudiantes
        ArrayList<String> estudiantes = new ArrayList<>();
        
        // Agregar 5 nombres de estudiantes al ArrayList
        estudiantes.add("Juan");
        estudiantes.add("María");
        estudiantes.add("Carlos");
        estudiantes.add("Ana");
        estudiantes.add("Luis");
        
        // Mostrar los estudiantes en el ArrayList
        System.out.println("Lista de estudiantes:");
        for (String estudiante : estudiantes) {
            System.out.println("- " + estudiante);
        }
        
        // Mostrar la cantidad de estudiantes
        System.out.println("\nTotal de estudiantes: " + estudiantes.size());
    }
}

File: C++/Ejercicio1.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

int main() {
    // Crear un vector de strings con 5 nombres de estudiantes
    vector<string> estudiantes;
    
    // Agregar 5 nombres de estudiantes al vector
    estudiantes.push_back("Juan");
    estudiantes.push_back("María");
    estudiantes.push_back("Carlos");
    estudiantes.push_back("Ana");
    estudiantes.push_back("Luis");
    
    // Mostrar los estudiantes en el vector
    cout << "Lista de estudiantes:" << endl;
    for (string estudiante : estudiantes) {
        cout << "- " << estudiante << endl;
    }
    
    // Mostrar la cantidad de estudiantes
    cout << "\nTotal de estudiantes: " << estudiantes.size() << endl;
    
    return 0;
}
